ansible/roles/products/teleop/server/desktop-icons/templates/scripts/wait_for_topic.j2

```diff
@@ -0,0 +1,17 @@
+#jinja2: trim_blocks:False
+#!/bin/bash
+
+# Arg1: Topic name to wait for
+# Arg2: Error message to display if topic doesn't appear until timeout
+elapsed_time=0
+start_time=$SECONDS
+while [ $elapsed_time -lt 60 ]; do
+  elapsed_time=$(( SECONDS - start_time ))
+  if [[ "$(docker exec -ti {{ container_name }} bash -c "source /home/user/projects/shadow_robot/base/devel/setup.bash;rostopic list")" = *"${1}"* ]]; then
+    # Success
+    exit 0
+  fi
+done
+zenity --error --text="${2}"
+docker stop {{ container_name }}
```

Comment:
this kind of break the single responsibility principle, could we please move the docker stop step outside the script into the scripts that call this one? (if exit code 1 -> docker stop)

Reply:
Moved in 3f7b0aafedb84a371589d2328d94e32c0e15b0dc.